### PR TITLE
added thshows_ids as a parameter to filter by path/tag

### PR DESCRIPTION
**Pull request by redglory**
Following the change in next episodes which can be filtered by path and tag parameters, I've applied same concept to nextairing and unaired episodes, on script.module.thetvdb module.

### CODE REVIEW

File: lib/thetvdb.py [hunk 1]
```diff
@@ -428,7 +428,7 @@ def get_kodishow_details(self, title, serie_details=None):
             result = serie_details
         return result
 
-    def get_kodi_unaired_episodes(self, single_episode_per_show=True, include_last_episode=False):
+    def get_kodi_unaired_episodes(self, single_episode_per_show=True, tvshows_ids=[], include_last_episode=False):
```
Comment (marcelveldt) [inline lib/thetvdb.py]: this is a breaking change... This module is also used in other projects.

Please make this additional parameter an optional filter param.
Also it's bad practice to set the default value to an empty array.

Please change the line to this:

def get_kodi_unaired_episodes(self, single_episode_per_show=True, include_last_episode=False, tvshow_ids=None):

File: lib/thetvdb.py [hunk 2]
```diff
@@ -437,17 +437,18 @@ def get_kodi_unaired_episodes(self, single_episode_per_show=True, include_last_e
...
-                episodes = [self.get_nextaired_episode(serieid)]
-            else:
-                episodes = self.get_unaired_episodes(serieid)
-            if include_last_episode:
-                episodes.append(self.get_last_episode_for_series(serieid))
-            for next_episode in episodes:
-                if next_episode:
-                    # make the json output kodi compatible
-                    next_episodes.append(self._map_kodi_episode_data(kodi_serie, next_episode))
+            if kodi_serie["tvshowid"] in tvshows_ids:
```
Comment (marcelveldt) [inline lib/thetvdb.py]: In relation to the previous comment, this line should be:

if not tvshow_ids or (tvshow_ids and kodi_serie["tvshowid"] in tvshows_ids):

**Review by marcelveldt — CHANGES_REQUESTED** [review]

**Comment by redglory** [comment]
Fixed tvshows_ids parameter and made it optional. :)

**Comment by marcelveldt** [comment]
Perfect, thanks !

**Pull request merged**